pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -30,8 +30,8 @@ import (
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/kubernetes/pkg/scheduler/algorithm"
 	priorityutil "k8s.io/kubernetes/pkg/scheduler/algorithm/priorities/util"
+	schedulerinternalpodinfo "k8s.io/kubernetes/pkg/scheduler/internal/podinfo"
```

Comment:
this import alias is pretty long, and we're already deep in the scheduler's implementation. I think it would be fine not to have an alias for this import.